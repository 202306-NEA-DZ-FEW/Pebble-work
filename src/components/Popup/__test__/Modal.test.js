import { render } from "@testing-library/react";
import React from "react";

import Modal from "../Modal";
describe("Modal", () => {
    it("renders correctly", () => {
        const { asFragment } = render(<Modal />);
        expect(asFragment()).toMatchSnapshot();
    });
});
