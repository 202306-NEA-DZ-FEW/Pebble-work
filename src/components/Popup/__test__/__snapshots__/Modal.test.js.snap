// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Modal renders correctly 1`] = `
<DocumentFragment>
  <div>
    <svg
      class="stroke-current shrink-0 h-6 w-6"
      fill="none"
      viewBox="0 0 24 24"
      xmlns="http://www.w3.org/2000/svg"
    >
      <path
        d="M9 12l2 2 4-4m6 2a9 9 0 11-18 0 9 9 0 0118 0z"
        stroke-linecap="round"
        stroke-linejoin="round"
        stroke-width="2"
      />
    </svg>
    <span />
  </div>
</DocumentFragment>
`;
